src/contracts/yas_pool.cairo

```diff
@@ -31,6 +35,17 @@ mod YASPool {
         unlocked: bool
     }
 
+    #[derive(Serde, Copy, Drop)]
+    struct ModifyPositionParams {
+        // the address that owns the position
+        owner: ContractAddress,
+        // the lower and upper tick of the position
+        tick_lower: i32,
+        tick_upper: i32,
```

Comment:
RJ created a struct that includes these attributes related to the position; could we change it to PositionKey? That's what I did in the update_position() because it made sense to me.
https://github.com/lambdaclass/yet-another-swap/blob/main/src/libraries/position.cairo#L19

Reply:
✅